# HG changeset patch
# User Rik <rik@octave.org>
# Date 1533837256 25200
#      Thu Aug 09 10:54:16 2018 -0700
# Node ID 0f6c81ca512899ed04302c69e96c20f7aab91a60
# Parent  134939b92d866564cb661e05164d64f5506c7dca
# Parent  2d5bd2cce0ddd581d2887b108572dd31d7b20a31
maint: merge stable to default.

diff --git a/scripts/general/flip.m b/scripts/general/flip.m
--- a/scripts/general/flip.m
+++ b/scripts/general/flip.m
@@ -15,37 +15,42 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <https://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {} flip (@var{x})
 ## @deftypefnx {} {} flip (@var{x}, @var{dim})
-## Flip array across dimension @var{dim}.
+## Return a copy of array @var{x} flipped across dimension @var{dim}.
 ##
-## Return a copy of @var{x} flipped about the dimension @var{dim}.
-## @var{dim} defaults to the first non-singleton dimension.
-## For example:
+## If @var{dim} is unspecified it defaults to the first non-singleton
+## dimension.
+##
+## Examples:
 ##
 ## @example
 ## @group
+## ## row vector
 ## flip ([1  2  3  4])
 ##       @result{}  4  3  2  1
 ##
+## ## column vector
 ## flip ([1; 2; 3; 4])
 ##       @result{}  4
 ##           3
 ##           2
 ##           1
 ##
+## ## 2-D matrix along dimension 1
 ## flip ([1 2; 3 4])
 ##       @result{}  3  4
 ##           1  2
 ##
+## ## 2-D matrix along dimension 2
 ## flip ([1 2; 3 4], 2)
 ##       @result{}  2  1
 ##           4  3
 ## @end group
 ## @end example
 ##
 ## @seealso{fliplr, flipud, rot90, rotdim, permute, transpose}
 ## @end deftypefn
